pkg/apis/ceph.rook.io/v1/labels.go

```diff
@@ -20,6 +20,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// SkipReconcileLabelKey is a label indicating that the pod should not be reconciled
+	SkipReconcileLabelKey = "ceph.rook.io/do-not-reconcile"
```

Comment:
FYI, the commit message and description of the PR say `ceph.rook.io/debug`.

Reply:
Good catch, I'll update it